projects/kubernetes/kubernetes/1-18/patches/0003-EKS-PATCH-staging-apiserver-disable-info-level-gRPC-.patch

```diff
@@ -5,7 +5,7 @@ Subject: [PATCH 03/16] --EKS-PATCH-- staging/*/apiserver: disable info level
  gRPC logging
 
 gRPC balancer wrapper has (non-leveled) info logging on "Notify" call.
-Which maens we are logging every single endpoint resolution.
```

Comment:
I'm going to deal with this in another PR so we can handle changes to patches in a more organized way